src/services/PodDatabase.ts

```diff
@@ -114,6 +120,17 @@ export default class PodDatabase {
       `memory=${info.memory} `)
   }
 
+  public getLifetimePodsUptime (): BigNumber {
+    let lifetimeUp = 0
+    for (let [_, value] of this.pods) {
```

Comment:
you could do `for (const value of this.pods.values())`, which might be more concise in this case